ee/vellum_ee/workflows/tests/test_server.py

```diff
@@ -135,16 +136,13 @@ class CodeExecutionNode(BaseCodeExecutionNode[BaseState, int]):
 
     # WHEN the workflow is loaded
     Workflow = BaseWorkflow.load_from_module(namespace)
-    workflow = Workflow()
+    workflow = Workflow(context=WorkflowContext(dynamic_file_loader=finder.loader))
 
     # THEN the workflow is successfully initialized
     assert workflow
 
     event = workflow.run()
-    assert event.name == "workflow.execution.fulfilled", event.model_dump_json()
-
-    # AND we pass in the correct file path to the open function
-    assert mock_open.call_args[0][0] == f"{namespace}/nodes/./script.ts"
+    assert event.name == "workflow.execution.fulfilled"
 
-    # AND we invoke the Code Execution Node with the correct code
-    assert vellum_client.execute_code.call_args.kwargs["code"] == ts_code
+    # AND we get the code execution result
+    assert event.body.outputs.final_output == 5.0  # type: ignore
```

Comment:
[nit] can avoid the type ignore for now with `event.body.outputs == {"final_output": 5.0}`